src/webextension/injections/js/bug1472075-bankofamerica.com-ua-change.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+/**
+ * Bug 1472075 - Build UA override for Bank of America for OSX & Linux
+ * WebCompat issue #2787 - https://webcompat.com/issues/2787
+ *
+ * BoA is showing a red warning to Linux and macOS users, while accepting
+ * Windows users without warning. From our side, there is no difference here
+ * and we receive a lot of user complains about the warnings, so we spoof
+ * as Firefox on Windows in those cases.
+ */
+
+/* globals exportFunction */
+
+if (!navigator.platform.includes("Win")) {
```

Comment:
This will cover Fennec as well, no? Do we want it to, when Fennec supports GoFaster? If not we might want to also add something like `&& navigator.userAgent.includes("Android")` just to future-proof it a bit.